src/gt4py/next/iterator/ir_utils/ir_makers.py

```diff
@@ -423,3 +423,28 @@ def as_fieldop(expr: itir.Expr, domain: Optional[itir.FunCall] = None) -> call:
             )
         )
     )
+
+
+def op_as_fieldop(
+    op: str | itir.SymRef | Callable, domain: Optional[itir.FunCall] = None
+) -> Callable[..., itir.FunCall]:
+    """
+    Promotes a function `op` to a field_operator.
+    `op` is a function from values to value.
+    Returns:
+        A function from Fields to Field.
+    Examples
+    --------
+    >>> str(op_as_fieldop("op")("a", "b"))
+    '(⇑(λ(__arg0, __arg1) → op(·__arg0, ·__arg1)))(a, b)'
```

Comment:
Please follow our coding conventions for docstrings  (google-style docstrings):
```suggestion
    Promotes a function `op` to a field_operator.
    
    Args:
        op: is a function from values to value.
    
    Returns:
        A function from Fields to Field.
        
    Examples:
        >>> str(op_as_fieldop("op")("a", "b"))
        '(⇑(λ(__arg0, __arg1) → op(·__arg0, ·__arg1)))(a, b)'
```